ide/extbrowser/src/org/netbeans/modules/extbrowser/NbDdeBrowserImpl.java

```diff
@@ -127,6 +133,59 @@ public NbDdeBrowserImpl (ExtWebBrowser extBrowserFactory) {
      */
     public static native String getDefaultOpenCommand() throws NbBrowserException;
     
+        /**
+     * Get the default browser name using Java JNA library
+     * @return String
+     */
+    private static String getDefaultWindowsBrowser() {
+        String userChoice = Advapi32Util
+                .registryGetStringValue(
+                        WinReg.HKEY_CURRENT_USER,
+                        "SOFTWARE\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\https\\UserChoice",
+                        "ProgId"
+                )
+                .toUpperCase(Locale.ROOT);
+
+        // done this way so that values like FirefoxURL-308046B0AF4A39CB can be handled
+        if (userChoice.toUpperCase().contains(ExtWebBrowser.FIREFOX)) {
+            return ExtWebBrowser.FIREFOX;
+        }
+        else if (userChoice.toUpperCase().contains(ExtWebBrowser.CHROME)) {
+            return ExtWebBrowser.CHROME;
+        } else if (userChoice.toUpperCase().contains(ExtWebBrowser.CHROMIUM)) {
+            return ExtWebBrowser.CHROMIUM;
+        } else if (userChoice.toUpperCase().contains(ExtWebBrowser.MOZILLA)) {
+            return ExtWebBrowser.MOZILLA;
+        } else {
+            return ExtWebBrowser.IEXPLORE;
+        }
```

Comment:
remove all `.toUpperCase()` from here since it is already in upper case at this point.

Reply:
When I backed up my fork before deleting and reforking netbeans I've somehow managed to grab the previous code configuration for this function :tired_face:

Now fixed again